layouts/partials/openapi/render-object-table.html

```diff
@@ -155,15 +155,17 @@
       adhering to a regex pattern
 
     * `anchor`: optional HTML element id for the target type, which will be used to link to it.
-
-    The title has a higher priority than anything else.
 */}}
 {{ define "partials/object-type-or-title" }}
     {{ $type := "object" }}
-    {{ if .title }}
+    {{ if and .title (or .properties (not (or .additionalProperties .patternProperties))) }}
```

Comment:
we should mention `.properties` in the comment for this partial.

Reply:
Done